api/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+const validate = require('../utils/validators/validator');
+const ctrlContact = require('../controller/controller');
+
+router.get('/contacts', ctrlContact.get);
+
+router.get('/contacts/:contactId', ctrlContact.getById);
+
+router.post('/contacts', () => validate.createContact, ctrlContact.addContact);
```

Comment:
() => validate.createContact - Testowałeś to ? Jak wywołujesz to przez funkcję to nie przekazujesz domyślnie parametrów i powinneś sam je podać.

wszędzie gdzie masz () => ......() powinneś przekazać req res i next czyli powinno to wyglądać tak:

(req, res, next) => validate.createContact(req, res, next),

to samo z resztą na dole. Przetestuj to potem czy napewno działa.

Reply:
Właśnie nie wiem czemu tak to zostawiłem. Miałem błąd w konsoli i testowałem rozwiązanie i tak zostało. 
Teraz już przesyłam sprawdzone, u mnie działa napewno! :P